packages/schematics/angular/README.md

```diff
@@ -0,0 +1,33 @@
+# @schematics/angular
+
+This package contains a collection of [schematics](/packages/angular_devkit/schematics/README.md)
+for generating an Angular application.
+
+# Schematics
```

Comment:
Shouldn't this be a subheading ie, `h2` ?

Reply:
done